python/ray/rllib/optimizers/async_samples_optimizer.py

```diff
@@ -85,7 +84,9 @@ def _init(self,
         self.timers = {k: TimerStat() for k in ["train", "sample"]}
```

Comment:
Does it also make sense to remove the timers?

Reply:
Well, I think so. Replaced self.timers with self._optimizer_step_timer.